### PR TITLE
lablqml.0.6: switching to dune

### PR DESCRIPTION
**Pull request by Kakadu**

### CODE REVIEW

File: packages/lablqml/lablqml.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+build: [
+  ["./configure"]
+  [make]
+]
+install: [make "install"]
+remove: [make "uninstall"]
+flags: [ light-uninstall ]
+depends: [
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/lablqml/lablqml.0.6/opam]: Do you use the Findlib library ? I couldn't see any reference of this in the source code. If so could you flag this dependency with the `{build}` attribute?

**Comment by kit-ty-kate** [comment]
Also there is a failure on Ubuntu 16.04 which seems related:
```
# startup_stubs.cpp: In function 'value caml_QGuiApplication_exec(value)':
# startup_stubs.cpp:198:19: error: 'nullptr' was not declared in this scope
#    Q_ASSERT(app != nullptr);
```
Something tells me you need to explicitly require for cpp11 or something

**Comment by Kakadu** [comment]
@kit-ty-kate I think that it's fine now

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**